app/App/Footer/RequestCommand/index.js

```diff
@@ -222,7 +217,7 @@ class RequestCommand extends React.Component {
     const isTestnet = this.store('main.networks', chain.type, chain.id, 'isTestnet')
     const {nativeCurrency, nativeCurrency:{symbol: currentSymbol = '?'}} = this.store('main.networksMeta', chain.type, chain.id)
     const nativeUSD = nativeCurrency && nativeCurrency.usd && !isTestnet ? nativeCurrency.usd.price : 0
-    // const value = this.hexToDisplayValue(req.data.value || '0x')
+    // const value = this.hexToDisplayValue(req.data.value || '0x') //TODO: if this is enabled in future, move the existing hexToDisplayValue into resources
```

Comment:
Do we need this?  If it's just a commented out thing I'd delete it...

Reply:
If we wont be adding it back in any time soon, happy to delete it! @floating ?